azdev/params.py

```diff
@@ -115,6 +116,20 @@ def load_arguments(self, _):
 
     # endregion
 
+    with ArgumentsContext(self, 'break-change command-meta export') as c:
+        c.positional('modules', modules_type)
+        c.argument('with_help', action="store_false", help="State whether to include help message")
+        c.argument('with_example', action="store_false", help="State whether to include examples")
+        c.argument('meta_output_path', help='command meta json file path to store')
+
+    with ArgumentsContext(self, 'break-change command-meta diff') as c:
+        c.argument('base_meta_path', required=True, help='command meta json file')
+        c.argument('diff_meta_path', required=True, help='command meta json file to diff')
+        c.argument('only_break', action="store_true", help='whether include non breaking changes')
+        c.argument('output_type', choices=diff_export_format_choices(), default=diff_export_format_choices()[0],
+                   help='format to print diff and suggest message')
+        c.argument('output_file', help='command meta diff json file path to store')
```

Comment:
You should consist with name of `meta_output_path`, `base_meta_path`, `diff_meta_path` and `output_file`. Using `_path` or `_file` subfix only. And do you need to add `meta` keyword in argument? It's already in command name.